FoodFriends/app/src/main/java/com/example/foodfriends/activities/SettingsActivity.java

```diff
@@ -0,0 +1,91 @@
+package com.example.foodfriends.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Switch;
+import android.widget.TextView;
+
+import com.example.foodfriends.R;
+import com.example.foodfriends.observable_models.UserObservable;
+import com.parse.ParseUser;
+
+import java.time.temporal.TemporalAccessor;
+import java.util.Observable;
+import java.util.Observer;
+
+public class SettingsActivity extends AppCompatActivity implements Observer {
```

Comment:
Why a settings activity? And not utilize a PreferenceFragment?

Reply:
I wanted it to take up the whole screen and not include navigation bar, so this was the simplest way to do it!